src/syscalls/syscall_response.rs

```diff
@@ -64,32 +67,46 @@ impl SyscallResponse {
 //   Response objects
 // ----------------------
 
+/// Gets the timestamp of the block in which the transaction is executed.
 #[derive(Clone, Debug, PartialEq)]
 pub struct GetBlockTimestampResponse {
+    /// The returned timestamp.
     pub timestamp: Felt252,
 }
 
+/// Deploys a new instance of a previously declared class.
 pub struct DeployResponse {
```

Comment:
same here. The documentation of this struct shouldn't be about what the corresponding syscall does. But about this Response struct.